src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx

```diff
@@ -408,8 +412,21 @@ function getDateFilterOption(
   return filterValue;
 }
 
-function formatDateValue(date: Date) {
-  return date.toISOString().slice(0, 10);
+function formatDateForTimezone(date: Date) {
```

Comment:
Can you use one of the date helpers in Quilt?
https://github.com/Shopify/quilt/tree/master/packages/dates

There is an helper to apply a timezone offset on a date. Maybe it could work?

Reply:
As discussed the date package is not available in Polaris.